pages/api/elements.py

```diff
@@ -676,8 +676,9 @@ def upload_image(self, request):
 
         location = request.data.get("location", None)
         is_public_asset = request.query_params.get('public', False)
+
         response_data, response_status = process_upload(
-            request, self.account, location, is_public_asset,
+            request, self.request.user, location, is_public_asset,
```

Comment:
This seems incorrect. The `account` is not always the `request.user`. The model is overriden in projects using this app through `settings.ACCOUNT_MODEL`